src/IKontrolCheatsBase.sol

```diff
@@ -31,10 +33,18 @@ interface KontrolCheatsBase {
     function setGas(uint256) external;
     // Returns a symbolic unsigned integer
     function freshUInt(uint8) external returns (uint256);
+    // Returns a symbolic unsigned integer with specified K variable name.
+    function freshUInt(uint8, string calldata) external returns (uint256);
```

Comment:
Would it make sense to add a similar variant for cheatcodes generating a fresh `uint` of a specific size, such as [this one](https://github.com/runtimeverification/kontrol-cheatcodes/blob/7baaf34ae72a11a6ba4e54021b1b3b7eb637e4ff/src/KontrolCheats.sol#L16)?
https://github.com/runtimeverification/kontrol-cheatcodes/blob/7baaf34ae72a11a6ba4e54021b1b3b7eb637e4ff/src/KontrolCheats.sol#L16

Reply:
Yes, I added these now. However I found I had to make the string arguments for those `memory` as opposed to `calldata`. I was wondering, should these all be `memory` type, including the ones in `IKontrolCheatsBase.sol`?